nvdaHelper/vbufBase/storage.h

```diff
@@ -266,6 +266,14 @@ class VBufStorage_fieldNode_t {
  */
 	bool addAttribute(const std::wstring& name, const std::wstring& value);
 
+/**
+ * Gets an attribute value for this field.
+ * @param name the name of the attribute
+ * @param value where to store the value for the attribute.
+ * @return true if the attribute exists, false if it doesn't exist.
+ */
+	bool getAttribute(const std::wstring& name, std::wstring* value);

```

Comment:
I don't know why all these comments don't match the identation, but it is unusual. I would assume there is no reason for it. Double check that doxygen generates docs and I believe it is fine

Reply:
Running DOxygen from NVDAHelper doesn't appear to build anything from docstrings of functions. I'm not sure what the expected output of DOxygen is, but it appears to just be generic notes written about *.dlls

Reply:
Hmm that doesn't sound right. Let's investigate this tomorrow.

Reply:
Thanks, I've added a reference to Doxygen in the relevant section of the readme.
It wasn't clear that I was looking to run `scons devDocs_nvdaHelper` based on the reference to Doxygen in the initial review comment.

Reply:
I have fixed the indentation on this PR, but the other items (with different indentation) also show up as expected in the generated documentation.